cloudformation/elk-stack.template

```diff
@@ -263,6 +263,11 @@
                         "Protocol": "TCP",
                         "LoadBalancerPort": "6379",
                         "InstancePort": "6379"
+                    },
+                    {
+                        "Protocol": "TCP",
+                        "LoadBalancerPort": "80",
+                        "InstancePort": "80"
```

Comment:
Personally, I like having all of the rules that describe the permissions for the internal ELB in one place ... but of course you are 100% correct that the description is now wrong. I will update it.